File: test/after/hello.js
const b = 0b111;
let a = 2 ** 3;
let undef;
const items = [1, 2, 3];
const obj = { a: 2, key: true, z: 1 };
const greeting = `hello ${a}`;
for (const item of items) {
  console.log(item);
}
items.map(x => x);
const fn = x => x;
if (a === 8) {
  a += b;
} else if (b) {
  a = b;
}
try {
  undef = fn(a);
} catch (ex) {
  console.log(ex);
}
if (!a) {
  console.log(obj, fn, undef, greeting);
}
const name = "test";
const o = {
  method() {
    if (a) {
      return true;
    }
    return false;
  },
  name,
};
console.log(o);
console.log(
  "argumentOne",
  "argumentTwo",
  "argumentThree",
  "argumentFour",
  "argumentFive",
  "six"
);
const ts = Date.now();
if (ts === undefined) {
  console.log(1, 2, 3);
}

File: test/after/hello.py
import os
import sys


class Foo:
    pass


class Bar(Foo):
    def __init__(self):
        super().__init__()


x = {}
y = []
z = ()
s = set()
f = frozenset()
d = {"b": 2, "a": 3}
v = f"{os.sep} {sys.path}"
if "a" in d:
    print("hello", x, y, z, s, f, d, v)
with open("file.txt") as fh:
    print("data", fh.read())


def long_function(
    argument_one, argument_two, argument_three, argument_four, argument_five, argument_six
):
    return argument_one
